components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/listener/IdentityOathEventListener.java

```diff
@@ -327,23 +330,27 @@ private boolean revokeTokens(String username, UserStoreManager userStoreManager)
                 // Also, if token binding is enabled, there can be multiple active tokens for the same user, scope
                 // and client combination.
                 // So need to revoke all the tokens.
-                return revokeTokens(accessTokens);
+                try {
+                    return revokeTokens(accessTokens);
+                } catch (IdentityOAuth2Exception e) {
+                    String errorMsg = "Error occurred while revoking Access Token";
+                    log.error(errorMsg, e);
+                    throw new UserStoreException(e);
+                }
             }
         }
         return true;
     }
 
-    private boolean revokeTokens(List<String> accessTokens) throws UserStoreException {
+    private boolean revokeTokens(List<AccessTokenDO> accessTokens) throws IdentityOAuth2Exception {
```

Comment:
Can we change the method signature to 
private boolean revokeTokens(AccessTokenDO[] accessTokens) ,
Which helps us not to create a List from Array

Reply:
This was leads to change [1] from List to Array.

[1] - https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1443/files/b28bfae4a96729a821fc14a3db4ade7333a520a1#diff-0daa59162506998f3129964e2307f192R334